### PR TITLE
Add new quote by William Butler Yeats 

### PR DESCRIPTION
**Pull request by tiresc**
This PR references #13 .

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -1,314 +1,318 @@
...
- {
-   "sentence": "Every language is a world. Without translation, we would inhabit parishes bordering on silence.",
-   "author": "Francis George Steiner"
- },
- {
-  "sentence": "We should learn languages because language is the only thing worth knowing even poorly.",
-  "author": "Kato Lomb"
-}
-]
+[{
```
Comment (jayehernandez) [inline server/data/quotes.json:1]: Hey @tiresc, thanks for contributing! Just have one quick comment.

Hope you can fix the formatting of this file using a formatter such as [this](https://jsonformatter.curiousconcept.com/) to lessen the number of commits (seems like the whole file was changed when only a few lines were added) 😅. Use **2 Space Tab** for the JSON template.

Comment (tiresc) [reply server/data/quotes.json]: Ok, I hopefully have fixed the formatting, I for some reason the file got messed up when I copied it from the JS linter.

Comment (jayehernandez) [reply server/data/quotes.json]: Thanks for fixing but it seems like it’s still detecting some changes that are not supposed to be added :( What quotes were you trying to add? Maybe I can help out with the formatting if it helps!

Comment (tiresc) [reply server/data/quotes.json]: Oh no, I'm not sure what I've changed to make it detect that. 
I wanted to add just one quote by Yeats.
"sentence": "Think like a wise man but communicate in the language of the people.",
"author": "William Butler Yeats"

Comment (jayehernandez) [reply server/data/quotes.json]: Fixed the spacing issue! :)

**Comment by jayehernandez** [comment]
Thanks for contributing @tiresc, working on some changes to add prettify and linting to avoid issues like this in the future!

@all-contributors please add @tiresc for content

**Comment by jayehernandez** [comment]
Oh no why 

@all-contributors please add @tiresc for content

**Pull request merged**